.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us fix the game
+
+---
+
+**Describe the bug**
+[A clear and concise description of what the bug is.]
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**System (please complete the following information):**
+ - OS: [e.g. Windows]
+ - Game Source: [e.g. Steam beta]
+ - Version: [e.g. 0.9.8 or commit hash cab74fd3]
+
+**Link to save file**
+[e.g. pastebin .com]
```

Comment:
just gonna suggest some non-cluttered services like hastebin or ghostbin and most importantly gists. pastebin isn't exactly well-suited for code.
also, is that space intentional?